openedx/core/djangoapps/catalog/management/commands/sync_course_runs.py

```diff
@@ -0,0 +1,52 @@
+"""
+Sync course runs from catalog service.
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.catalog.utils import get_catalog_course_runs
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
One of the most important things about moving towards a world of microservices, is observability.  Can you ensure that you have enough logging in place to allow you to monitor/debug issues in production?  And, can you ensure that your log statements are easily searchable in splunk (via a common prefix, for example)?